test/framework/easyconfigs/yeb/goolf-1.4.10.yeb

```diff
@@ -3,7 +3,7 @@ _internal_variables_:
 
     - &comp_name GCC
     - &comp_version 4.7.2
-    - &comp [*comp_name, *comp_version]
+    - &comp !join [*comp_name, ', ', *comp_version]
```

Comment:
this looks very weird now...

maybe it's better to drop `*comp`, and just use `*comp_name, *comp_version` below?

or use this here (does that work?):

```
&comp *comp_name, *comp_version
```